test/integration/admin_can_view_orders_test.rb

```diff
@@ -1,6 +1,13 @@
 require "test_helper"
 
 class AdminCanViewOrdersTest < ActionDispatch::IntegrationTest
+  test "when an admin logs in they see the admin dashboard" do
+    create_admin
+    login_admin
+
+    assert_equal '/admin/dashboard', current_path
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.